### ISSUE TITLE
Linter TOC + ' used without matching

### ISSUE THREAD

**Issue by cosydney** [sindresorhus/awesome-lint#152]
When I run: 
```npx awesome-lint https://github.com/axolo-co/awesome-slack-apps```

I get a few errors. 

I don't understand this one: 
- ToC item "Standup Slack app bot" does not match corresponding heading "Code related integrations"

And the ' used without matching seems to be a false positive. 

<img width="1129" alt="image" src="https://user-images.githubusercontent.com/10532381/191695984-9ea44bae-e115-4c24-bd78-ae7dc8399277.png">

**Comment by cosydney** [comment]
I found out some of the errors, was using ’ instead of '

**Comment by rtbs-dev** [comment]
I just had this error too, I feel your pain. That teaches me to copy from a PDF I guess. :sweat_smile: 

Could be helpful to lint/warn against certain nasty characters like `’` and the weird space characters, etc.
